028-HubAndSpoke/Coach/01-HubNSpoke-basic.md

```diff
@@ -0,0 +1,19 @@
+# Challenge 1: Building a basic Hub And Spoke topology 
+
+**[Home](README.md)** - [Next Challenge >](./02-AzFW.md)
+
+## Introduction
+
+In this challenge you will be setting up a basic hub and spoke topology with connectivity to onprem via VPN site-to-site.

```

Comment:
If there isn't a coach specific Introduction, then you can delete this whole section.

Reply:
Added more details to the intro